pom.xml

```diff
@@ -25,6 +25,7 @@
         <apache-httpclient-fluent.version>4.5.13</apache-httpclient-fluent.version>
         <apicurio.registry.utils.serde.version>1.3.2.Final</apicurio.registry.utils.serde.version>
         <quarkiverse.apicurio.registry.client.version>0.0.2</quarkiverse.apicurio.registry.client.version>
+        <confluent.kafka-avro-serializer.version>6.0.0</confluent.kafka-avro-serializer.version>
```

Comment:
The latest version is `6.2.0`, please update it here otherwise Dependabot will create a separated PR for doing only this.